utils.go

```diff
@@ -175,12 +181,18 @@ func logVersions() {
 	log.Println("VERBOSE: Helm client version: " + helmVersion)
 }
 
+// add $$ escaping for $ strings
+func expandEnv(s string) string {
+	os.Setenv("HELMSMAN_DOLLAR", "$")
```

Comment:
Why not do this directly in the `substituteEnv` function bellow?

Reply:
No specific reason. Both can be used.